apps/dot_ext/views/authorization.py

```diff
@@ -20,10 +19,8 @@ class AuthorizationView(DotAuthorizationView):
     use the custom AllowForm.
     """
     form_class = SimpleAllowForm
-    login_url = getattr(
-        settings, 'AUTHORIZATION_LOGIN_URL', "/mymedicare/login")
-    template_name = getattr(
-        settings, 'AUTHORIZATION_TEMPLATE_NAME', "design_system/authorize.html")
+    login_url = "/mymedicare/login"
```

Comment:
This should be a setting so it looks like the update in `apps/mymedicare_cb/views.py`

Reply:
Okay, I understand the confusion. There's a key difference there, which is that those URLs are actually environment specific. They point to another server, and the environments need to match (more or less). This references a location on the local server, which is not environment specific. So the other one needs to be a setting, but this one doesn't.

Reply:
Alternative answer - this should be reverse('mymedicare-login'), which is what this actually is